packages/admin/src/brevoContacts/form/BrevoContactForm.tsx

```diff
@@ -174,10 +179,17 @@ export function BrevoContactForm({ id, scope, input2State, additionalFormFields,
                     <MainContent>
                         <Box sx={{ marginBottom: 4 }}>
                             <Alert severity="warning">
-                                <FormattedMessage
-                                    id="cometBrevoModule.brevoContact.contactEditAlert"
-                                    defaultMessage="Editing a contact will affect all scopes and the target groups within those scopes."
-                                />
+                                {mode === "edit" ? (
+                                    <FormattedMessage
+                                        id="cometBrevoModule.brevoContact.contactEditAlert"
+                                        defaultMessage="Editing a contact will affect all scopes and the target groups within those scopes."
+                                    />
+                                ) : (
+                                    <FormattedMessage
+                                        id="cometBrevoModule.brevoContact.contactAddAlert"
+                                        defaultMessage="The contact will get a DOI email to confirm the subscription. After the contact's confirmation, the contact will be added to the corresponding target groups in this scope depending on the contact's attributes. Before the confirmation the contact will not be shown on the contacts page."
```

Comment:
I would write DOI out:
"Double Opt-In" or "DOI(Double Opt-In)"

Not all users will know what Double Opt-In is and even less will know the abbreviation DOI.
At least with Double Opt-In you can easily google what it is.

Reply:
You are right. I change `DOI` to `double opt-in` in lower case. Since brevo is using the lowercase version in their explanation. https://www.brevo.com/blog/double-opt-in/

Reply:
[ec28a03](https://github.com/vivid-planet/comet-brevo-module/pull/47/commits/ec28a03d296ef5af7b586a9b977b3aa9c453484c)